### PR TITLE
Dreamcast Redump Supplement Launch

### PR DESCRIPTION
**Pull request by SmokeMonsterPacks**

### CODE REVIEW

File: EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+c880c653be7b03b1e069c51744f0312961594117849f16c9bfc7f55de992599a	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Dream Selection Vol 1 (PD) DC Evolution 2002/Dsvol1.cdi	1c12e0c5b148df3aed4811698eafd04a325975b9	3dbeeb783261fd16893fc46853213ce8	5ac1b0fc
+361658e5bf2022e5d18d7ad404e7073c8101e86781735554343c544419669cea	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Drill (PD) Handmade Videogames 2005-09-05/Drill001.cdi	3b2bd50d3bd961c3a345f6172728cc0a31422834	4f99b49da73cd63d9a63782d75e41697	9b461ce1
+2471e346a3d5616df3ee3970485ac3b0edc92e7caff1b623b08d5f530287efcb	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Epitech Homebrew Collection (PD) Epitech Class of 2002/Epitech.cdi	7efa2930e9b86d3f474052c5a03ebc19e9b5a045	5a8590f822926dd962e07cf53484d90c	fb59d06e
+990c264be1074c81acf55d527ee38b72604ad2ba241bdc2baccec0acee791cc7	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Epitech Homebrew Collection (PD) Epitech Class of 2002/image1.JPG	969e04efdb3f6c7443f4cbca7609db501e8cce58	fffcf3730094ccb36ff7d4296916e046	2ebf1a70
```
Comment (tjanas) [inline EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt:17]: should a JPG be included?

File: EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+e817d25cb733c9d0af855e88373f8fa211c208183fb66b36dc77d5ab29beb72c	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Beats of Rage Collection 3 (PD) DCEvolution/RIPFIRE_BORC3.cdi	309423a1fbc241747adce464ed7a7934d0cec726	98bac011f8d7400a73456acae7672dbd	6daf4321
+36bc140502d73a8f5baf716233620c43306ff781677b4cd7f859c01f1ee28e60	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Beats of Rage Double Dragon Collection - Revolution, Revolution 2, DD3 (PD)/Double Dragon Collection.cdi	e5869ddb5f7d2c26994db68a425f8ea3b22afc14	df921b02a66f19852c63bb58d27133f0	eea2cbd7
+fa72f5e1c893d13304eb8a140531ed056c9575ef61a958f1cdffe97e92364466	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Capcom 10-in-1 (Hack) v2 Toodles TDC/TDCFinal2.cdi	a5c964446b3e418d68c27d2f4b8ddfaadba0c9a1	53a33efa0de17e48fdab31fcdb0d5ca5	fe9918e2
+ab80fd3e4ad379ab98879e488abee180924f86bf85646751c7e915b812bb8f8b	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Cave Story (PD) NXDC 2017-05-25/NXDC.mdf	37a55e33e73550a5cfd42f81c00d26ae86630968	1db4fae90bbbae80db0a08187a374f29	8cf1c918
```
Comment (tjanas) [inline EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt:9]: not sure if mdf/mds images are compatible with gdemu or MODE

File: EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+e949841427edb3bc14b25ecd2da69ab6e1e742d8f6ec05679c54dec1bed0849c	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/FlashBack - port (PD) v0.1.8 REminiscence 2005/FlashBack (REminiscence) (PD) v0.1.8 (port) 2005.cdi	c0a5f3d9039efd928dd0b647d68169e593e6345d	23463c20964970cb0d36da5d99aa7716	dc153294
+3f909f050e9a6850927234bee1dc01e7613c27cd9c67fff46108a12c9a555dde	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Fruit'Y (PD) v1 Retroguru 2015/fruity.cdi	1e87147aa76829ded212fe6197f19912061c1054	2326c2be39dbc28ff27772a8e0be0294	52ebc50e
+3c60c0757647dca05167e027fb2b951ec795b9621d046ad0be18fadedc476cdb	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Giana's Return (PD) v1.10.1 Retroguru 2015/gr-v110b-dc.cdi	3e7279cee7b9e4d0d74267f681c79d54b423a3bb	233e296f937721b2a06493bd0ad1af8c	5d7555c0
+9bf4fdca73ed1f9654ba5930261fd8d31d3874327e9925bf64c4ac7dfd9ac11a	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Giana's Return (PD) v1.10.1 Retroguru 2015/docs/libmodplug/AUTHORS	12bb23885f7389df3e2327dfdf5c23c3d9e68a42	9399482837bdb3e5a608f9f2180996cc	6b8b87f4
```
Comment (tjanas) [inline EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt:21]: remove docs subdirectory?

File: EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+bb117c0fbd7f57f64170b690285d7df07c2371b578e3b3cd3aa2e1155ef461a0	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Giana's Return (PD) v1.10.1 Retroguru 2015/docs/sdl_mixer/COPYING	720ac006232639ed551ce48d638dee35f8d378d4	27818cd7fd83877a8e3ef82b82798ef4	6e078824
+e1eb3a6ed298c488f5210e89b8ad924166d5e8b1ce1ff3f91b8d5cfc520e3a0c	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Giana's Return (PD) v1.10.1 Retroguru 2015/docs/sdl_mixer/README	8625583ec8e9e3ae47aa6ca895ecaccd3449cae5	43c42cf76cb9f8384e0cfea5fe671652	e5af316f
+1a09937f90583e12e43aeba0e5ed422b77bc5a879a481c9dbf37b3f3e880b38b	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Giana's Return (PD) v1.10.1 Retroguru 2015/docs/zlib/README	cedff0e7ab9fabb2fec8baac4ee16cc892b2a963	944a28bd770ac46f1a0995f5fba5d83c	5d212e55
+a25e2276f92c43d69c10499b2aa53cfcd3bb5e7419254fb31ae7d72bd2d33978	Dreamcast Redump Supplement/@Homebrew, Unlicensed, Etc/Hydra Castle Labyrinth - port (PD) E. Hashimoto 2020/Hydra Castle Labyrinth SRC.7z	818d7be0d6d8353dfeda566383fdc9d001542296	69bee6dbd5595aefa968efb303d31e08	d9fa22fc
```
Comment (tjanas) [inline EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt:29]: 7z files shouldn't be in an SMDB

File: EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+57eff24181c0581e969a40ec4972336da0891dcd093f814d7dd577c64e0142a6	Dreamcast Redump Supplement/@Restorations, Improvements, Video Hacks/Resident Evil - Code Veronica NTSC+VGA (Hack) v1 japanese_cake/Resident Evil Disc 2 NTSC+VGA (Hack) v1 japanese_cake/track02.raw	b643df4def042b6c23f90f54c690ac7df431233f	d1838bcbca4a24b39deddeae2a9d371a	15e73429
+6bda6b0ee667c845ecc7a572b448f8ffee08cf5decc7920ce43c566c5779d7d5	Dreamcast Redump Supplement/@Restorations, Improvements, Video Hacks/Resident Evil - Code Veronica NTSC+VGA (Hack) v1 japanese_cake/Resident Evil Disc 2 NTSC+VGA (Hack) v1 japanese_cake/track03.bin	8a19ac6f38e43840f9783f00c5a51ff0f4367920	08d80cd93ae0623e0c29852f7697fec1	fd6f4d3f
+0c1ca0d1fada11e3e76e333729ebc965c23262bab69d45f70667538991a5f219	Dreamcast Redump Supplement/@Restorations, Improvements, Video Hacks/Skies of Arcadia NTSC+VGA (Hack) v1 japanese_cake/Skies of Arcadia v1.000 (2001)(Sega)(PAL)(M4)(Disc 1 of 2)[!][f VGA-60hz japanese_cake].gdi	2a37adc5e75d6f43b27bc61dc4fc09d5ad473ce0	9d1f603062285fdb7bc099ed4d1e5ff1	b220115c
```
Comment (tjanas) [inline EverDrive Pack SMDBs/Dreamcast Redump Supplement.txt:75]: I see the gdi and bin/raw files for Disc 1, but not Disc 2?

**Comment by frederic-mahe** [comment]
Nice review Terry!

**Comment by SmokeMonsterPacks** [comment]
Thanks, I'll clean this up. MDF is supported.

**Pull request merged**